docs/guides/troubleshooting.md

```diff
@@ -155,3 +155,48 @@ usually found in the following locations, should they need to be deleted:
 $HOME/Library/Logs/CoreSimulator/*
 $HOME/Library/Developer/Xcode/DerivedData/*
 ```
+
+## A gesture, like scroll or swipe, does not have any effect / It is unclear how to do it
```

Comment:
Would suggest a shorter title - even simply 'Gestures' would be fine for a dedicated document

Reply:
I have added it to troubleshooting because people often complain their gesture does not work and this piece describes how to make it work

Reply:
Sure, but the troubleshooting part is only half of this piece, so I'd say it's better to have a separate document here. The list of relevant execute methods in particular is a very useful reference in its own right.

Reply:
moved to a separate document